_posts/2022-09-03-aphp-user-story.adoc

```diff
@@ -0,0 +1,74 @@
+- 100 000 health professional taking care of our patients - including nearly 12 200 doctors, 4000 interns, over 53 000 nurses, paramedic and social workers
+- Around 2 000 volunteers working with patients and their families and over 1400 helping with legal issues
+- 3 500 research project in various fields, 650 active international patent portfolios
+- In our staff one doctor out of five was formed directly by the AP-HP in collaboration with 7 medical schools.
+- 34 schools, including 16  nurse training center (IFSI)
+- A 7.8 billions of euros budget
+- Nearly 1,5 emergencies and over 2 million calls to the 15 (French emergency number)
+- round 38 000 births in our 13 maternities
+- 315 000 surgical procedures
+- 1 800 transplants
+- 63 500 COVID patients
+- 68 cooperation agreements in Asia, Middle East, Africa and South America
+
+== DSN/DEV
+
+Direction des Services Numériques/DEVeloppement
+
+The DSN/DEV  is in charge of developing and maintaining various apps and specific tools. The IT department is as vast as the APHP itself. Dealing with every single medical specialty, it is sometimes difficult to find every needed tool on the market. That’s where the DSN/DEV intervene, creating ultra specialized micro apps. 
+With all these heterogeneous apps, the APHP faces multiple challenges in terms of communication. Once again it is the DSN/DEV job to ensure that every single part is able to communicate properly with each other, regardless of their nature and function. In doing so their job naturally requires mastery over the data flux and the ability to manipulate it in real time.
+
+
+== Our Chosen Technologies
+
+Historically a vast panel of technologies were used. In the last 10 years, our efforts were oriented toward unification. Today we are currently using only two languages : some PHP development projects for historical reasons and for the rest we are working with Java EE/Angular.
+
+For that last part, our choices leaned toward WildFly/JBoss. We chose to package our applications in a single deliverable containing front end and back end due to the small size and to facilitate deployment. Our front end is rarely evolving, or to be more exact it evolves only when the back end is doing so. This allows our teams to quickly deploy new features. The size of our Information Network (hundreds of major apps and thousands of working instances) makes it so its global evolution is slow paced.
+
+Using containers made us revise our choices. A key point in our decision was to keep working with technologies already mastered by our teams, and thus ensuring that any deployment or development of an app wouldn’t imply a total start from scratch, but rather the use of pre-existing mods. We wanted an evolution and not a revolution.
+
+== Quarkus
+
+The DSN/DEV kept close attention to Quarkus while researching other technologies as well. With the release 1.x of Quarkus, it became evident that it would become the tool that would take us in the right direction. 
```

Comment:
```suggestion
The DSN/DEV kept close attention to Quarkus while researching other technologies as well. With the 1.x release of Quarkus, it became evident that it would become the tool that would take us in the right direction. 
```